spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -57,10 +57,17 @@ public LocalModuleDeployer(ModuleLauncher launcher) {
 	@Override
 	public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		String module = request.getCoordinates().toString();
+		if (request.getCount() != 1) {
+			logger.warn(String.format("%s only supports a single instance per module; ignoring count=%d for %s",
```

Comment:
Yes, and that's one more reason not to go down the path of multiple module instances for *local* deployment yet (if at all). In a containerized/cloud deployment ports are mapped.